aegis-web/yo/app/index.html

```diff
@@ -102,9 +103,11 @@
     <link rel="stylesheet" href="styles/jupyter-tool.css">
     <link rel="stylesheet" href="styles/tools-jupyter-overview.css">
     <link rel="stylesheet" href="styles/leaflet-areaselect.css">
+    <link rel="stylesheet" href="styles/ng-tags.custom.css">
     <!-- endbuild -->
   </head>
   <body  ng-app="hopsWorksApp">
+    <div growl></div>
```

Comment:
Just curious. what is that?

Reply:
This is needed to display popup notifications to the user, e.g. whether saving form data was successfully or failed. The code already used the growl library for that, but notifications would not display without this required div.